docs/demultiplexing.md

```diff
@@ -160,6 +160,52 @@ Example configuration file for pipeline with `gather_from_node` specified before
 }
 ```
 
+## Dynamic batch handling with demultiplexing
+
+Demutliplexing feature enables handling requests with dynamic batch size. For workloads that involve frequent batch size change this is recommended way to handle such requests.
+In order to use it pipeline config has to have additional field demultiply_count. To leverage this feature requests, and responses have to have additional dimension of inputs/outputs.
+
+In the case of Resnet 50 with standard input (1,3,224,224) and N images, the inputs and outputs should have following shapes
```

Comment:
In the case of ResNet-50 model with shape (1,3,224,224) and N images, the inputs and outputs will be like presented below: